### ISSUE TITLE
Prefix/suffix: Add better documentation on how to use this correctly

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#2299]
Prefix and suffix is in the way it is implemented hidden from screen readers. The reason for this is that the same information should be available in the label of the input field. 

In our storybook there is now an example with prefix/suffix without label and description of how to use it at the bottom. The example and text should be more connected and explicit in how it is used. If they do not include a label with the same information.

**Comment by mimarz** [comment]
UK GOV have a good way of describing correct use https://design-system.service.gov.uk/components/text-input/
